mono/metadata/class.c

```diff
@@ -3381,6 +3381,21 @@ mono_class_is_subclass_of (MonoClass *klass, MonoClass *klassc,
 	return result;
 }
 
+static gboolean 
+mono_interface_implements_interface (MonoClass *interface_implementer, MonoClass *interface_implemented)
+{
+	int i;
+	MonoClass **klass_interfaces = m_class_get_interfaces (interface_implementer);
```

Comment:
AOT tests are failing.  So probably we need `mono_class_setup_interfaces (interface_implementer, error)`